.github/ISSUE_TEMPLATE/NCO_bug_report.yml

```diff
@@ -1,6 +1,7 @@
 name: NCO Bug Report
 description: Report something that is incorrect or broken
 labels: ["nco-bug", "triage"]
+type: ["bug"]
```

Comment:
Is `type` supposed to be a string rather than a list?

Reply:
I think you're right